addons/hr/static/tests/m2x_avatar_employee.test.js

```diff
@@ -67,6 +67,7 @@ test("many2one in list view", async () => {
     expect(".o_avatar_card").toHaveCount(1);
     expect(".o_avatar_card_buttons button:eq(0)").toHaveText("Send message");
     await contains(".o_avatar_card_buttons button:eq(0)").click();
+    await waitFor(".o-mail-ChatWindow");
     expect(".o-mail-ChatWindow").toHaveCount(2);
     await waitFor(".o-mail-ChatWindow-header:contains('Luigi')");
```

Comment:
The assertion is for 2 chat windows, I support the `waitFor(".o-mail-ChatWindow")` could resolve before the 2nd chat window is shown.

An already fix is to just swap the 2 LOCs `expect()` and `waitFor()` below.